src/command-parser/parsers.ts

```diff
@@ -39,8 +42,16 @@ export const parseBIP32Path = (
   throw Error(Errors.InvalidPathError)
 }
 
-export const parseFileTypeMagic = (fileTypeMagic: string, path: string) => {
+export const parseFileTypeMagic = (fileTypeMagic: string, pathType: PathTypes) => {
   if (fileTypeMagic.startsWith('Payment')) {
+    // TODO cardano-cli doesn't support 'Mint' prefix now, but maybe it will
+    if (pathType === PathTypes.PATH_WALLET_MINTING_KEY) {
```

Comment:
Maybe we can write it as a switch based on pathType? Because it seems to me that once pathType is PATH_WALLET_MINTING_KEY, fileTypeMagic.startsWith('Payment') must be true (i.e. we only assert it, not make a decision based on it).